src/resources/date-utilities.ts

```diff
@@ -0,0 +1,353 @@
+import { isDate } from 'lodash'
```

Comment:
I'd rather have this in a new folder. We've been using "resources" for resource files.

Reply:
Would it make sense to call these files "services" and put them in the services folder?

Reply:
Yes, I wondered about the correct directory to place these files.  Any objection if I create a new directory called `utilities`?

Reply:
Go for it.

Might want to rename "utils" folder though...?

Reply:
I like your idea of putting them in the existing "services" folder.